eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -45,6 +45,7 @@
                 <source>..\..\appconfiguration\client\src\main\java</source>
                 <source>..\..\appconfiguration\client\src\samples\java</source>
                 <source>..\..\core\azure-core\src\main\java</source>
+                <source>..\..\core\azure-core\src\samples\java</source>
```

Comment:
If you don't add this, do you still need to have spotbugs exclusions?

Reply:
No you don't.

Reply:
Then I would recommend not including snippet code for spotbugs inspections.

Reply:
But we should add this line here so that spotbugs would run for other samples.